test/test_examples_2d_old.jl

```diff
@@ -346,15 +352,6 @@ end
   end
 end
 
-
-# Only run extended tests if environment variable is set
-if haskey(ENV, "TRIXI_TEST_EXTENDED") && lowercase(ENV["TRIXI_TEST_EXTENDED"]) in ("1", "on", "yes")
-  @testset "Examples (long execution time)" begin
-    @test_nowarn Trixi.run(joinpath(EXAMPLES_DIR, "parameters_euler_blob_shockcapturing_mortar.toml"))
-    @test_nowarn Trixi.run(joinpath(EXAMPLES_DIR, "parameters_euler_ec_mortar.toml"))
```

Comment:
Where did this test go? Was this already moved to the regular tests?

Reply:
Yes. I've moved `parameters_euler_blob_shockcapturing_mortar.toml` in this PR and `parameters_euler_ec_mortar.toml"` was moved (with smaller final time) some time ago.